test/unit/generator/content/part2.test.js

```diff
@@ -0,0 +1,101 @@
+const part2 = require('../../../../app/generator/content/part2')
+const mockStatement = require('../../../mock-statement-data')
+
+describe('generate part 2', () => {
+  test('includes header', () => {
+    const result = part2(mockStatement)
+    expect(result[0].text).toBe('Part 2. Calculation')
+  })
+
+  test('includes introduction with scheme short name', () => {
+    const result = part2(mockStatement)
+    expect(result[1].text).toBe(`We calculated the total ${mockStatement.scheme.shortName} payment amount by adding together payments for all the standards which are part of your agreement, as shown in this table.`)
+  })
+
+  test('includes all table columns', () => {
+    const result = part2(mockStatement)
+    expect(result[2].table.body.length).toBe(6)
+  })
+
+  test('includes all table columns', () => {
```

Comment:
test names aren't descriptive enough, these have the same name but 2 different results

Reply:
The one above is a copy of one below without the correct name.  Fixed.